Extensions/bezier/bezier.lisp

```diff
@@ -287,53 +283,29 @@ second curve point, yielding (200 50)."
 (defgeneric segments (design))
 
 (defmethod segments ((design bezier-design))
-  (let ((tr (transformation design)))
-    (mapcar (lambda (s) (transform-segment tr s)) (%segments design))))
+  (%segments design))
 
 (defmethod transform-region (transformation (design bezier-design))
-  (let* ((tr (transformation design))
-         (result (if (translation-transformation-p transformation)
-                     (make-instance (class-of design)
-		       :segments (%segments design)
-		       :transformation 
-		       (compose-transformations transformation tr))
-                     (make-instance (class-of design)
-		       :segments (mapcar (lambda (s)
-					   (transform-segment transformation s))
-				  (segments design))))))
-    (when (translation-transformation-p transformation)
-      (setf (original-region result) (or (original-region design) design)))
-    result))
+  (make-instance (class-of design)
+                 :segments (mapcar (lambda (s)
+                                     (transform-segment transformation s))
+                                   (segments design))))
 
 (defmethod compute-bounding-rectangle* ((design bezier-design))
-  (multiple-value-bind (lx ly ux uy) (call-next-method)
-    (let ((tr (transformation design)))
-      (transform-rectangle* tr lx ly ux uy))))
+  (call-next-method))
 
```

Comment:
good point. I think I just had this in there because I didn't want to go through the hassle of trying to delete that specific method,